addons/arsenal/CfgACEArsenalStats.hpp

```diff
@@ -0,0 +1,96 @@
+class CfgACEArsenalStats {
+    class statBase {
+        scope = 1;
+        priority = 0;
+        stats[]= {};
+        displayName= "";
+        showBar= 0;
+        showText= 0;
+        barStatement = "";
+        textStatement = "";
+        condition = "true";
+        tabs[]= {{}, {}};
+    };
+    class bananaPotassium: statBase {
```

Comment:
We should prefix those too, this config will be public API.

I think it's best to prefix all, but some base ones that have nothing to do with ACE features could be left unprefixed.

Reply:
what kind of prefix tho.

Reply:
`ACE_`